.github/workflows/ci.yml

```diff
@@ -56,10 +63,3 @@ jobs:
           name: node-pty-${{ matrix.name }}-${{ matrix.arch }}
           path: node_modules/@lydell/node-pty-${{ matrix.name }}-${{ matrix.arch }}
           if-no-files-found: error
-      - name: Upload node-pty-${{ matrix.name }}-${{ matrix.arch2 }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: node-pty-${{ matrix.name }}-${{ matrix.arch2 }}
-          path: node_modules/@lydell/node-pty-${{ matrix.name }}-${{ matrix.arch2 }}
-          if-no-files-found: error
-        if: ${{ matrix.arch2 != 'none' }}
```

Comment:
This is unfortunately still needed for the Windows build, to upload the Windows arm64 build.

Reply:
True! The previous iteration had a separate runner for Windows arm64, and there was no need for this anymore.